docs/install.md

```diff
@@ -13,7 +13,7 @@ cd pori
 ## Start the Authentication Server
 
 Before any of the other systems can be set up you will need to start the authenication server. By
-default, PORI authenticates against an instance of KeyCloak. For convenience we have provided a
+default, PORI authenticates against an instance of [KeyCloak](https://www.keycloak.org/). For convenience we have provided a
```

Comment:
the line length here is inconsistent

Reply:
we're not enforcing a max line length for markdown. I am trying to remember to do all new content without breaking paragraphs though